### PR TITLE
API Customer: CDR resource

### PR DESCRIPTION
**Pull request by workgena**
- [x] CDR resource for Customer API
- [x] documentation

### CODE REVIEW

File: db/secondbase/structure.sql [hunk 1]
```diff
@@ -375,7 +375,7 @@ CREATE TABLE cdr (
     yeti_version character varying,
     lega_user_agent character varying,
     legb_user_agent character varying,
-    uuid uuid,
+    uuid uuid DEFAULT public.uuid_generate_v1() NOT NULL,
```
Comment (dmitry-sinina) [inline db/secondbase/structure.sql]: Так не нужно делать. Генерация uuid происходит в хранимке.

File: app/models/cdr/cdr.rb [hunk 2]
```diff
@@ -167,6 +167,8 @@ class Cdr::Cdr < Cdr::Base
 
   scope :account_id_eq, lambda { |account_id| where('vendor_acc_id =? OR customer_acc_id =?', account_id, account_id) }
 
+  scope :where_customer, -> (id) { where(customer_id: id) }
+  scope :where_account, -> (id) { where(customer_acc_id: id) } # OR vendor_acc_id ???
```
Comment (workgena) [inline app/models/cdr/cdr.rb]: @dmitry-sinina take a look at this query. This is ony for Customers API, so I did not add `OR vendor_acc_id = ?`, I'm not sure about this.

File: app/resources/api/rest/customer/v1/cdr_resource.rb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  model_name 'Cdr::Cdr'
+
+  key_type :uuid
+  primary_key :uuid
+
+  attributes :time_start,
+             :time_connect,
+             :time_end,
+             :duration,
+             :success, # TODO: or it should be "status"?
```
Comment (workgena) [inline app/resources/api/rest/customer/v1/cdr_resource.rb]: In Admin-panel this field replaced with `status` (text). But in API i think it should be closer to real data. @dmitry-sinina what do you say?

File: app/resources/api/rest/customer/v1/cdr_resource.rb [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+             :dst_prefix_in,
+             :to_domain,
+             :ruri_domain,
+             :diversion_in,
+             :local_tag,
+             :lega_disconnect_code, # TODO: or it shold be "leg_a_dc"
+             :lega_disconnect_reason, # TODO: or it should be "leg_a_reason"
+             :lega_rx_payloads,
+             :lega_tx_payloads,
+             :auth_orig_transport_protocol_id, # TODO: relation "auth_orig_transport_protocol"?
```
Comment (workgena) [inline app/resources/api/rest/customer/v1/cdr_resource.rb]: @dmitry-sinina perhaps it should be replaced with relation `auth_orig_transport_protocol`?

Comment (dmitry-sinina) [reply app/resources/api/rest/customer/v1/cdr_resource.rb]: yes

File: app/models/cdr/cdr.rb (hunk 2 above)
Comment (dmitry-sinina) [reply app/models/cdr/cdr.rb]: ` { where(customer_acc_id: id) }` is right.

File: app/resources/api/rest/customer/v1/cdr_resource.rb (hunk 3 above)
Comment (dmitry-sinina) [reply app/resources/api/rest/customer/v1/cdr_resource.rb]: I think "success" is good for api

**Pull request merged**